lua/nvim-treesitter/configs.lua

```diff
@@ -325,6 +326,10 @@ function M.setup(user_data)
       if config[mod].keymaps and type(data.keymaps) == 'table' then
         config[mod].keymaps = data.keymaps
       end
+      if mod == 'ensure_installed' then
+        config.ensure_installed = data
+        require'nvim-treesitter/install'.ensure_installed(data)
+      end
```

Comment:
i think as this function will only run once, we don't need to add it to `config.ensure_installed`.
Though i wonder if you had something in mind by storing this in the config object ?

Reply:
not, really. I only thought it would be strange if all other data gets stored and this one not.

Reply:
hum well remove it for now, i'll do a small refactoring to make sure this can be stored properly.

Reply:
ok. it's removed.